Utilities/Utilities/_Rating/PointsRating.swift

```diff
@@ -7,37 +7,160 @@
 //
 
 import Foundation
+import StoreKit
 
+/// See notion https://www.notion.so/dydx/App-Store-Review-Spec-226c9031b0c746e488e02c7eeb1ab46f?pvs=4
 open class PointsRating: NSObject, RatingProtocol {
-    public func add(points: Int) {
-        self.points = self.points + points
-    }
+    
```

Comment:
The new logic is specific to the V4 app, so it's better to create a new implementation of RatingProtocol within the V4 framework folder.  We can keep the original PointsRating implementation here, since it's generic/app-agnostic.